ci/scripts/benchmark.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+# Exits as soon as any line fails.
+set -euo pipefail
+
+while getopts 's:' opt; do
+    case ${opt} in
+        s )
+            SKU=$OPTARG
+            ;;
+        \? )
+            echo "Invalid Option: -$OPTARG" 1>&2
+            exit 1
+            ;;
+        : )
+            echo "Invalid option: $OPTARG requires an argument" 1>&2
+            ;;
+    esac
+done
+shift $((OPTIND -1))
+
+function polling() {
+    set +e
+    try_times=30
+    while :; do
+        if [ $try_times == 0 ]; then
+            echo "❌ ERROR: Polling Timeout"
+            exit 1
+        fi
+        psql "$@" -c '\q'
+        if [ $? == 0 ]; then
+            echo "✅ Instance Ready"
+            break
+        fi
+        sleep 10
+        try_times=$((try_times - 1))
+    done
+    set -euo pipefail
+}
+
+function cleanup {
+  echo "--- Delete tenant"
+  rwc t delete -tenant ${TENANT_NAME}
+}
+
+trap cleanup EXIT
+
+if [[ -z "${RISINGWAVE_IMAGE_TAG+x}" ]]; then
+  IMAGE_TAG="latest"
+else
+  IMAGE_TAG="${RISINGWAVE_IMAGE_TAG}"
+fi
+
+date=$(date '+%Y%m%d-%H%M%S')
+TENANT_NAME="bench-${date}"
```

Comment:
are we going to use the same format of tenant name for single and multinode benchmark runs?